web/yo/app/scripts/controllers/gene.js

```diff
@@ -2764,7 +2777,6 @@ angular.module('oncokbApp')
                 });
                 var deferred6 = $q.defer();
                 firebase.database().ref('Meta/' + $routeParams.geneName).on('value', function (doc) {
-                    $scope.geneStautMessage = 'Last edit was made on ' + new Date(doc.val().lastModifiedAt) + ' by ' + doc.val().lastModifiedBy;
```

Comment:
No longer needed?

Reply:
No, I asked Jiaojiao and she said  this should be removed.